routes/users.js

```diff
@@ -1,9 +1,60 @@
 var express = require('express');
 var router = express.Router();
+var passport = require('passport');
+const db = require('../models');
+
+//This file needs to run and have nothing to export
+require('../controllers/user.controller');
 
 /* GET users listing. */
 router.get('/', function(req, res, next) {
   res.send('respond with a resource');
 });
 
+router.get('/auth/facebook', passport.authenticate('facebook', { scope : ['email']}));
+router.get('/auth/facebook/check', (req,res) => {res.send('everything is fine')});
```

Comment:
care to explain why is this needed?

Reply:
the facebook/check is just sanity check leftovers. removed it